src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html

```diff
@@ -1,4 +1,5 @@
-<tm-loading-retry [shouldShowRetry]="!coursePresent" [message]="'Failed to load course'" (retryEvent)="getCourseEnrollPageData(courseId)">
+<tm-loading-retry [shouldShowRetry]="isLoading" [message]="'Loading course...'" (retryEvent)="getCourseEnrollPageData(courseId)">
```

Comment:
I think the more serious problem here is that you should not use `tm-loading-retry` to show the loading message.

Reply:
Thanks David, sorry initially I was coming from the angle that we wanted to keep the retry button while loading